src/Kiota.Builder/Plugins/AdaptiveCardTemplate.cs

```diff
@@ -0,0 +1,120 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Text.Json;
+using System.Text.Json.Serialization;
+using Microsoft.Extensions.Logging;
+
+namespace Kiota.Builder.Plugins
+{
+
+    internal class AdaptiveCardTemplate
+    {
+        private readonly ILogger<KiotaBuilder> Logger;
+        private string AdaptiveCard => @"
```

Comment:
this should be an embedded resource instead to ease up review/updates.

https://stackoverflow.com/a/41106851/3808675